#global testerror "Err0238"

func titi() throw
{
    throw 0
}